i18n/messages-en.xtb

```diff
@@ -372,4 +372,14 @@
 	<translation id="831441529070502458" key="MSG_PET_SET_DETAIL_PODS_DESIRED_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="The message says that that many pods are desired to run (pet set details page)."><ph name="PODS_COUNT" /> desired</translation>
 	<translation id="1739547616641575970" key="MSG_PET_SET_LIST_CREATED_AT_TOOLTIP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Tooltip 'Created at [some date]' showing the exact creation time of pet set.">Created at <ph name="CREATION_DATE" /></translation>
 	<translation id="1719241183992408191" key="MSG_PET_SET_LIST_PET_SET_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label 'Pet Set' which will appear in the pet set delete dialog opened from a pet set card on the list page.">Pet Set</translation>
-</translationbundle>
+	<translation id="7594899852613234459" key="MSG_NAMESPACE_NOT_SELECTED" source="/usr/local/google/home/bryk/src/github.com/dashboard/.tmp/serve/app-dev.js" desc="Text for dropdown item that indicates that no namespace was selected">all namespaces</translation>
```

Comment:
I suggest `All namespaces` or `All Namespaces` depending on the general guideline uses (sentence case vs. title case).